.github/workflows/licensed.yml

```diff
@@ -0,0 +1,54 @@
+name: licenses check
+
+on:
+  push:
+    branches:
+      - main
+    tags-ignore: "**"
```

Comment:
If you want to ignore pushes to tags, the `branches` filter above should be sufficient.

Reply:
done in https://github.com/actions/actions-sync/pull/94/commits/fc6c0a105afef7c6474259884fe2ad40d5453f0d